### DISCUSSION TITLE
Milvus查询性能与服务器核数关系问题

### DISCUSSION THREAD

**Discussion by jie2050** [milvus-io/milvus#5136] [Q&A and General discussion]
场景描述：
1、milvus版本：1.0
2、测试服务器信息：CentOS Linux release 7.5.1804 (Core)，98G内存，16核和32核各一台，cpu型号：Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
3、入库量：1亿，nlist：1024，index_file_size：4096；topk : 200，nprobe : 64，nq：4
4、压测工具：jmter5.3，并发数：2
问题描述：
1、测试脚本和并发数不变的情况下，只改变milvus服务器的cpu核数，在低核数的性能会比高核性能更优？且服务器的CPU使用率都达到95%左右。
16核服务器的压测性能指标：
聚合报告：
![image](https://user-images.githubusercontent.com/35087871/117429433-a2032f80-af59-11eb-813b-85ee594c6d1d.png)
服务器负荷情况：
![image](https://user-images.githubusercontent.com/35087871/117429548-be9f6780-af59-11eb-83d7-8e591e8aee2d.png)
32核服务器的压测性能指标：
聚合报告：
![image](https://user-images.githubusercontent.com/35087871/117429638-d37bfb00-af59-11eb-8c2a-6eda4accc472.png)
服务器负荷情况：
![image](https://user-images.githubusercontent.com/35087871/117429692-df67bd00-af59-11eb-9e39-91875b9940f7.png)

**Comment by yhmo** [comment]
"16核和32核各一台，cpu型号：Intel(R) Xeon(R) CPU E5-2670"
从Intel官网上看，这款cpu是8核16线程，跟你说的你的“16核和32核”不一致，这两台机器是否一样的cpu？

**Reply by jie2050** [reply]
这边是在VMware ESXi虚拟的2台服务器。测试过程中，cpu型号是一致，但核数不同。
![image](https://user-images.githubusercontent.com/35087871/117521466-842cdd80-afe0-11eb-9543-aa9623ad417f.png)

**Comment by shengjun1985** [comment]
Could you provide the two debug logs for us to analyze the time cost of each step?
Could you show the outputs of the command `lscpu` on the two machines?
